lib/kasket.rb

```diff
@@ -39,4 +39,17 @@ def self.cache_store=(options)
   def self.cache
     @cache_store ||= Rails.cache
   end
+
+  def self.pending_records
+    Thread.current['kasket_pending_records']
```

Comment:
I saw symbols more often and it makes more sense to me to not waste gc ...

On Wed, Jan 6, 2016 at 3:29 PM, Gary Grossman <notifications@github.com>
wrote:

> In lib/kasket.rb
> <https://github.com/zendesk/kasket/pull/31#discussion_r48936077>:
>
> > @@ -39,4 +39,17 @@ def self.cache_store=(options)
> >    def self.cache
> >      @cache_store ||= Rails.cache
> >    end
> > +
> > +  def self.pending_records
> > +    Thread.current['kasket_pending_records']
>
> I had it that way, then I changed it to be consistent with the other uses
> of Thread.current
>
> https://github.com/zendesk/kasket/blob/master/lib/kasket/configuration_mixin.rb#L9-L17
>
> I could change them all to use symbols...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/kasket/pull/31/files#r48936077>.
>